index.js

```diff
@@ -149,19 +124,19 @@ function createKeySigner(bits) {
     // Even though we are specifying "RSA" here, this works with ECDSA
     // keys as well.
     var signer = crypto.createSign('RSA-SHA' + bits);
```

Comment:
the `RSA-` prefix can be dropped from all these calls and just use the digest names `sha256` and the like.